app/src/organizers/models.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from users.models import User
+from conferences.models import Zosia
+
+def validate_phone_number(value):
+    phone_reg = r'\d{9}'
+    m = re.match(phone_reg, value)
+    if not m:
+        raise ValidationError(_('Please provide correct phone number'))
+
+class OrganizerContact(models.Model):
+    user = models.OneToOneField(
+        User,
+        on_delete=models.CASCADE,
+        primary_key=True,
```

Comment:
I wouldn't specify it as primary key. To me the auto-generated one should suffice.